### PR TITLE
when uid is long, trim to avoid PathTooLong filename

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class SplitClassPageToMemberLevel : BaseDocumentBuildStep
         private const char Separator = '.';
         private const string SplitReferencePropertyName = "_splitReference";
         private const string IsOverloadPropertyName = "_isOverload";
+        private const int MaximumFileNameLength = 180;
```
Comment (vwxyzh) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs]: Can we disable it?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs (hunk 1 above)
Comment (vicancy) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs]: No, we cannot. Plan to remove this when FAL is enabled.

**Pull request merged**